apps/public/src/app/nieuwe-renderer/page.tsx

```diff
@@ -0,0 +1,19 @@
+'use client'
+
+// eslint-disable-next-line import/order
+import { Heading } from '@amsterdam/design-system-react'
+
+// eslint-disable-next-line import/order
+import { FormRenderer } from '@meldingen/form-renderer'
+import { Grid } from '@meldingen/ui'
+
+const Bedankt = () => (
```

Comment:
Even though it's dummy code. But confusing that the page component is called "Bedankt"